src/puppetlabs/puppetdb/command.clj

```diff
@@ -97,8 +97,8 @@
 
 (def supported-command-versions
   {"replace facts" (version-range 2 4)
-   "replace catalog" (version-range 4 7)
-   "store report" (version-range 3 6)
+   "replace catalog" (version-range 4 8)
```

Comment:
I think it was probably removed when we retired all the old commands, when we put them back again we didn't put back the deprecated handling: https://github.com/puppetlabs/puppetdb/blob/2.3.x/src/com/puppetlabs/puppetdb/command.clj#L323